components/SparcCard/SparcCard.vue

```diff
@@ -37,36 +37,62 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+$image-fixed-large: 350px;
+$image-fixed-small: 250px;
+$tablet-large: 1024px;
+$tablet-small: 768px;
 .sparc-card {
-  @media (min-width: 768px) {
+  @media (min-width: $tablet-small) {
     display: flex;
   }
   &--image-right {
     .sparc-card__image {
-      @media (min-width: 768px) {
+      @media (min-width: $tablet-small) {
         order: 2;
       }
     }
   }
   &__image {
-    background-size: cover;
-    background-position: center;
-    height: 206px;
-    @media (min-width: 768px) {
-      height: auto;
+    @media (min-width: $tablet-large) {
+      flex: 0 0 $image-fixed-large;
+      max-height: $image-fixed-large;
+    }
+    @media (min-width: $tablet-small) and (max-width: $tablet-large - 1) {
+      flex: 0 0 $image-fixed-small;
+      max-height: $image-fixed-small;
+    }
+    img {
```

Comment:
Make the images `display: block` to remove the extra space underneath.
```suggestion
    img {
      display: block;
```